ur_bringup/launch/ur_control.launch.py

```diff
@@ -154,6 +154,18 @@ def launch_setup(context, *args, **kwargs):
             "stdout": "screen",
             "stderr": "screen",
         },
+        condition=IfCondition(use_fake_hardware),
+    )
+
+    ur_control_node = Node(
```

Comment:
Maybe a short comment would be good here, as otherwise things might get a bit confusing for newcomers.